ipwb/replay.py

```diff
@@ -553,7 +553,7 @@ def all_exception_handler(error):
 # the future.
 @app.route('/ipwbconfig/<requestedSetting>')
 def getRequestedSetting(requestedSetting):
-    return Response(ipwbUtils.getIPFSAPIHostAndPort() + '/ipwbassets')
+    return Response(ipwbUtils.getIPFSAPIHostAndPort() + '/webui')
```

Comment:
The comment above this block is misleading and the `requestedSetting` parameter is unused.

Reply:
Yeah, you are right. We used to use (or perhaps just anticipated to use) the route for additional setting retrieval. I will re-check on this and make it a bit more semantic.